projects/kubernetes/Makefile

```diff
@@ -18,21 +22,19 @@ push-container-images:
 
 build-vm-images: kube-master.iso kube-node.iso
 
-# NB cannot use $^ because $(INIT_YAML) is not for consumption by "moby build"
-kube-master.iso: kube.yml $(KUBE_RUNTIME).yml $(KUBE_RUNTIME)-master.yml $(INIT_YAML)
-	moby build -name kube-master -format iso-efi -format iso-bios kube.yml $(KUBE_RUNTIME).yml $(KUBE_RUNTIME)-master.yml
+kube-master.iso: kube.yml $(KUBE_RUNTIME).yml $(KUBE_RUNTIME)-master.yml $(KUBE_NETWORK).yml
+	moby build -name kube-master $(KUBE_FORMAT_ARGS) $^
 
-kube-node.iso: kube.yml $(KUBE_RUNTIME).yml
-	moby build -name kube-node -format iso-efi -format iso-bios $^
+kube-node.iso: kube.yml $(KUBE_RUNTIME).yml $(KUBE_NETWORK).yml
+	moby build -name kube-node $(KUBE_FORMAT_ARGS) $^
 
-network.yaml: $(KUBE_NETWORK).yaml
-	ln -nf $< $@
+weave.yml: kube-weave.yaml
 
-weave-%.yaml:
-	curl -L -o $@ https://cloud.weave.works/k8s/v1.8/net?v=$*
+kube-weave.yaml:
+	curl -L -o $@ https://cloud.weave.works/k8s/v1.8/net?v=v2.0.5
```

Comment:
can we put the weave version at the top of the file? Hard to find down here

Reply:
Sure.

I had debated supporting `KUBE_NETWORK="weave v2.0.5"` like syntax and playing games like `$(first $(KUBE_NETWORK)).yml` to get `weave.yml` and `net?v=$(word 2,$(KUBE_NETWORK))` for the URL. Seemed overly clever to me though.